### DISCUSSION TITLE
Email confirmation links/Magic links not working with Outlook

### DISCUSSION THREAD

**Discussion by smilo7** [supabase/supabase#3961] [Questions]
I am having issues with confirmation emails and magic links when used from outlook/hotmail accounts.

When you click the link in the email, it sends the user to this address:
```https://mysite.com/#error_code=404&error_description=Confirmation+Token+not+found```
for magic links ```Confirmation+Token+not+found``` changes to ```User+not+found```

If the link is copy pasted instead, there is no error and you arrive logged in at the site.

I believe the issue is that mimecast is processing the link in a way that breaks it.

has any one else encountered this issue and/or knows a solution?

**Comment by jopfre** [comment]
I also have an issue with Mimecast intercepting links and redirecting them through their own domain first which leads to error=unauthorized_client&error_code=401&error_description=email+link+is+invalid+or+has+expired in the url

**Comment by carvuk22** [comment]
same issue here

**Comment by cdimitroulas** [comment]
I believe the reason this happens is that the way these "Safe Links" features (e.g. in Outlook) work is by sending a HEAD request to the link to get some information from it.

The way to fix it would be for Supabase to ignore HEAD requests and only invalidate the magic link after a proper GET request.

**Reply by Draxeytina** [reply]
How do you set that up?

**Reply by cdimitroulas** [reply]
I don't believe there is any way to do so, it would need to be something implemented in Supabase itself.

**Comment by drummerjolev** [comment]
I'm facing the same issue. Any chance this can be fixed soon?

**Edit**: [this is a known issue](https://supabase.com/docs/guides/auth/auth-email-templates#limitations)

**Comment by carlosvirreira** [comment]
Same issue here

**Comment by nboaldin** [comment]
I am creating workarounds for this but its a pain in the butt. Would love a cleaner resolution... or at least called out in the auth flow docs (if its not already).

**Comment by rami-maalouf** [comment]
@nboaldin @carlosvirreira @drummerjolev I found a simple fix which I shared in this YouTube video: https://youtu.be/fkUPlmiNPoc
